composer.json

```diff
@@ -10,10 +12,10 @@
     ],
     "require": {
         "php": ">=7.1",
-        "vimeo/psalm": "^4.8",
```

Comment:
no, I move psalm in `require-dev`

Reply:
Why did you mode it to a dev dependency?

Reply:
@AdeAttwood because some not so smarty ide's parse composer.json "as is" - if it's in production section then package must be loaded

Reply:
I don't think we need to think about IDE's, I think we need to think about what the `require` is for. This package depends on psalm to be installed and will not work without it. Because of this I think we will leave it in the `require` section.

Reply:
@AdeAttwood Psalm don't used in production, however other plugin require psalm in prod section - ok, rollback